### PR TITLE
menu & localization & slide homepage

### PR DESCRIPTION
**Pull request by HoaiPEC**
## Related Tickets:
- [27683](https://edu-redmine.sun-asterisk.vn/issues/27683)

## Notes Impacted Areas:
- N/A.

## Attachments:
homepage-menu

### CODE REVIEW

**Comment by HoaiPEC** [comment]
ready

File: resources/views/client/homepage/index.blade.php [hunk 1]
```diff
@@ -4,17 +4,14 @@
...
     <section id="home-section" class="hero">
         <div class="home-slider owl-carousel">
-            <div class="slider-item">
+            <div class="slider-item" style="background-image:url({{ asset(config('path-img.slider-1')) }})">
+                <div class="overlay"></div>
+            </div>
+            <div class="slider-item" style="background-image:url({{ asset(config('path-img.slider-2')) }})">
+                <div class="overlay"></div>
+            </div>
+            <div class="slider-item" style="background-image:url({{ asset(config('path-img.slider-3')) }})">
```
Comment (huyenttm-0901) [inline resources/views/client/homepage/index.blade.php:13]: viết riêng css ra file riêng

Comment (HoaiPEC) [reply resources/views/client/homepage/index.blade.php]: e sửa rồi ạ

**Comment by HoaiPEC** [comment]
ready

**Pull request merged**